### DISCUSSION TITLE
Possession Mod follows invisible phantom camera.

### DISCUSSION THREAD

**Discussion by 1-Mythical** [winstxnhdw/lc-hax#64] [Ideas]
What if instead of directly controlling everything about the monsters, you would guide them instead. This would mean that the normal ai would be used except it's goal is to get to your camera. So the camera should be able to go through walls. For example, if I were to possess a Bracken, my camera can move around, but I wouldn't actually move the Bracken. Then the Bracken would sneak towards my camera location if it can reach it. This would fix a lot of debugging with recreating controls for certain monsters because they use their default movement speed and behavior ai except the goals of where it should go should be where the players camera is.

**Comment by winstxnhdw** [comment]
Currently, this is the default for Bunker Spider. We can implement it for all but it'll be a lot of work so don't expect it any time soon.

**Reply by 1-Mythical** [reply]
I completely understand. I will wait as long as needed as this is a cool feature.

**Reply by jezbud** [reply]
how exactly do you use possess?

**Reply by 1-Mythical** [reply]
Right click on an enemy and press ```z``` to exit and ```x``` to toggle realistic movement.
